src/main/antlr4/Manifold.g4

```diff
@@ -12,8 +16,9 @@ WHITESPACE: [ \t\r\n]+ -> skip;
 
 INTEGER_VALUE: [0-9]+;
 BOOLEAN_VALUE: 'false' | 'true';
+TYPE_KEYWORD: 'Type';
 
-tupleTypeValueEntry: (IDENTIFIER ':')? typevalue (':' expression)?;
+tupleTypeValueEntry: (IDENTIFIER ':')? typevalue ('=' expression)?;
```

Comment:
This changes the syntax for setting a default value for a tuple entry from
```
(foo:Int:7)
```

to
```
(foo:Int = 7)
```

I think the latter is much more sane.